pebble_tool/commands/sdk/create.py

```diff
@@ -97,7 +97,9 @@ def __call__(self, args):
             ])
 
             if args.javascript:
-                file_list.extend([('app.js', 'src/js/app.js'), ('pebble-js-app.js', 'src/js/pebble-js-app.js')])
+                file_list.extend([
```

Comment:
This change is going to end poorly for anyone using an SDK other than the most recent beta version.